.drone.yml

```diff
@@ -0,0 +1,126 @@
+pipeline:
+
+  test:
+    image: node:10
+    secrets:
+      - npm_auth_token
+    commands:
+      - npm ci
+      - npm test
+    when:
+      event: [push, pull_request, tag]
+
+  compile:
```

Comment:
It's be good to get an [audit step](https://github.com/UKHomeOffice/asl/blob/master/.drone.yml#L13-L20) in here too, but not critical at this point.